### DISCUSSION TITLE
List all available tags from Docker hub

### DISCUSSION THREAD

**Discussion by sFritsch09** [docker/extension-ideas#171] [Ideas]
Instead of going to the browser and search for available **image tags** on **docker hub** ==>

Better have an **extension** to search for images and their available tags **sorted by the latest version**:

For golang image:

```bash
curl -sL 'https://registry.hub.docker.com/v2/repositories/library/golang/tags?page_size=1024' | jq '."results"[]["name"]' | sort -Vr
```

and then you can **copy the tag** to use it in your Dockerfile or in your docker run command

**Comment by ajeetraina** [comment]
@sFritsch09 Docker Desktop comes with Global Search Capability by default. You can search by typing a Docker Image name and it displays you all the available tags right in the Dashboard. Give it a try!!

<img width="1330" alt="Screenshot 2023-06-23 at 12 11 24 PM" src="https://github.com/docker/extension-ideas/assets/313480/16ee07f1-d493-4090-b3ad-46dc48e49ce1">

**Reply by sFritsch09** [reply]
Oh nice I totally missed that, thanks 🚀
